docs/source/apps.rst

```diff
@@ -76,7 +76,7 @@ To get help configuring a new App, use:
 
 .. code-block:: bash
 
-  $ python manage.py app new --help
+  python manage.py app new --help
```

Comment:
Whatsup!!! Why do you want to remove `$ ` that was added in previous PR #1114 couple days ago?

Reply:
IMO, I think in this case we can get rid of it. The `$` is intended to signal that a code block is specifically **_bash_**, but here we already have the `code-block:: bash` declaration. 

Additionally... `$` isn't really all that relevant anymore, since macos ZSH uses `%`............

Reply:
yeah I got ride of it for this very reason - shell prompts are arbitrary so putting `$` in here seems unnecessary and more of a "personal" preference. I as also unable to find any documented uses of `.. code-block:: bash` that used prompts